docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2034,39 +2034,39 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  When we are not able to get the _dir_ to add servlet for service _key-value_.
+  When we are not able to get the dir to add servlet for service _key-value_.
 
   **Action:**
 
-  Check if your _dir_ is exists and is valid.
+  Check if your dir is exists and is valid.
 
 
 
 ### ZWED0081W
 
-  Could not start tomcat, error=_errorMessage_
+  Could not start Tomcat, error=_errorMessage_
 
   **Reason:**
 
-  Tomcat manager is unable to start tomcat with _DOptionsArray_.
+  Tomcat manager is unable to start Tomcat with DOptionsArray.
```

Comment:
What is DOptionsArray? This needs more user friendly description

Reply:
Doesn't need to be very descriptive but hint: look at what DOptionsArray is composed of